docs/BigQueryCreateDataset-action.md

```diff
@@ -0,0 +1,56 @@
+# BigQuery Create Dataset
+
+#### **Description**
+
+Creates a BigQuery Dataset.
+
+#### **Usage**
+
+Dataset names must be unique per project. The dataset name contain up to 1,024 characters and can
+contain letters (upper or lower case), numbers and underscore. Dataset names cannot contain special characters
+such as '-', '&', '*', or '%'.
+
+If dataset name starts with '_' (underscore) then the dataset would be hidden and not visible in
+the view and dataset names are case sensitive (E.g. 'dataset' and 'Dataset' are two different
+datasets that can co-exists in a project.
+
+#### **Basic**
+
+Following are properties used to configure this plugin
```

Comment:
Does this sentence fragment really convey anything useful? IMO it doesn't and can be removed. If we really want to keep it, we would want it to be exactly the same in every reference doc and it should be:

The following properties can be used to configure this plugin: